cluster-scope/base/rbac.authorization.k8s.io/clusterroles/curator-operator-fetchdata-editor-role/clusterrole.yaml

```diff
@@ -0,0 +1,24 @@
+# permissions for end users to edit fetchdata.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: curator-operator-fetchdata-editor-role
+rules:
+- apiGroups:
+  - curator.operate-first.io
```

Comment:
Isn't the group that you have defined `curator.operatefirst.io` rather than `curator.operate-first.io`?

Reply:
good catch!

Reply:
Yes, this should be curator.operatefirst.io. As this is auto-generated when we do make generate and make manifest. 
And in all the roles it was correct. I don't know how this got missed. Nice catch. Will update it in the next commit.